app/helpers/ontology_version_helper.rb

```diff
@@ -0,0 +1,32 @@
+module OntologyVersionHelper
+  def failed?(ontology_version)
+    ontology_version.state == 'failed'
+  end
+
+  def latest_version?(ontology_version)
+    ontology_version.ontology.versions.last == ontology_version
```

Comment:
I agree that this is the most robust version and we should stick with it.

Also, I think we could use this as a method of the model `OntologyVersion`.